mewarpx/mewarpx/assemblies.py

```diff
@@ -308,6 +309,7 @@ def __init__(self, z, V, T, WF):
         # set label to correctly grab scraped particles from the buffer
         self.scraper_label = 'z_hi'
 
+
```

Comment:
I assume this is from auto-formatting changes? If not, probably better to leave out.

Reply:
No I added that. Our standard is to have 2 lines between `class` definitions but these were lacking, so just catching things up to standard.